PROCESS.md

```diff
@@ -0,0 +1,119 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+# OpenColorIO Project Processes
+
+This document outlines important development and administrative processes and 
+procedures adhered to by the OpenColorIO project. It should be reviewed in 
+conjuntion with [CONTRIBUTING.md](CONTRIBUTING.md) prior to proposing or 
+submitting changes.
+
+* [Core Library Changes](#Core-Library-Changes)
+* [Required Approvals](#Required-Approvals)
+
+## Core Library Changes
+
+This procedure outlines the OpenColorIO project's process for design, 
+development, and review of changes to the core library. The intent is for 
+discussion and decisions around such changes to be made in public and have 
+transparent outcomes, seeking the guidance of both the TSC (Technical Steering 
+Commitee) and the wider OCIO community and stakeholders.
+
+**NOTE:** Changes to OpenColorIO process or governance are considered core 
+library changes since they have cascading effects across the project.
+
+1. All core changes begin with community discussion, which can involve informal 
+   conversation in [Slack](http://slack.opencolorio.org/) or on 
+   [ocio-dev](https://lists.aswf.io/g/ocio-dev), but are ultimately brought to a 
+   TSC meeting where concensus can be reached around the proposal's relevance 
+   and approach. TSC meetings are public and meeting notes are posted to the 
+   [OCIO repo](https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/ASWF/meetings/tsc) 
+   within days of a meeting for a permanent record of decisions.
+2. Following acceptance of the proposal by the TSC, an issue is created in 
+   GitHub to present a design proposal. A period of further community 
+   discussion is encouraged around the proposed changes, captured in issue 
+   comments. Ongoing concerns may continue to be raised in this issue, and 
+   brought to the TSC prior to and during development.
+3. Feature proposal issues are added to the 
+   [Roadmap](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3) 
```

Comment:
Not totally opposed, but I find the "Roadmap" name potentially misleading for people.  A "roadmap" is generally used to define a release or a set of releases.  The GitHub Project as described here is only going to be populated as new features are proposed and accepted.  There is no clear mapping of these to a given release.  It may also give the impression that the roadmap is more empty or closer to completion than is actually the case.  Perhaps a name such as "Feature development" or something like that would be more clear?

Reply:
Good point. Renamed to `Feature Development`: https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3